app/jobs/duplicate_revisions_job.rb

```diff
@@ -0,0 +1,10 @@
+class DuplicateRevisionsJob < ApplicationJob
+  def perform(item_id)
+    item = Item.find(item_id)
+    original_item_revisions = ItemRevision.where(item_uuid: item.duplicate_of).pluck(:revision_uuid)
```

Comment:
We should probably add some sort of authentication here to prove that the duplicate_of belongs to the same user. Otherwise a client can put any duplicate_of UUID and the server will obey.